protected/humhub/modules/admin/views/group/edit.php

```diff
@@ -20,15 +20,20 @@
             'form' => $form,
             'model' => $group,
             'attribute' => 'defaultSpaceGuid',
-            'selection' => $group->defaultSpace,
-            'maxSelection' => 1,
+            'selection' => $group->defaultSpaces,
+            'maxSelection' => 1000,
         ])
         ?>
     <?php endif; ?>
 
+    <div class="reassign-spaces">
+        <?= Html::a(Yii::t('AdminModule.user', 'Reassign All'), Url::to(['/admin/group/reassign-all', 'id' => $group->id]), ['class' => 'btn btn-primary btn-reassign', 'data-method' => 'POST', 'data-confirm' => Yii::t('AdminModule.modules', 'Are you sure? Reassign default spaces to all users?')]); ?>
```

Comment:
Are you sure? All members of this group will be reassigned to all specified Spaces.

Reply:
Done!

Reply:
Done!